backend/tests/integrated/infrastructures/load/adapters/test_repositories.py

```diff
@@ -0,0 +1,137 @@
+import pytest
+from django.db import DatabaseError
+from pytest_mock import MockFixture
+
+from infrastructures.apps.common import exceptions as common_exceptions
+from infrastructures.apps.load import models
+from modules.common import pagination as pagination_dtos
+from modules.load.domain import ports, value_objects
+from modules.load.services import queries
+from modules.load.services.queries import ports as query_ports
+from modules.load.services.queries import queries as query_dtos
+from modules.load.services.queries.ports import repositories as query_repositories
+from tests.model_factories import DataFactory
+
+from .....fakers import fake_transformed_data
+
+
+def test_django_data_domain_repository_create_method_creates_record(
+    test_django_data_domain_repository: ports.AbstractDataDomainRepository,
+):
+    # Given
+    data = [fake_transformed_data()]
+
+    # When
+    test_django_data_domain_repository.create(data)
+
+    # Then
+    assert all(
```

Comment:
Actually, there is a small difference in a check to be done.

1) In Given section please specify how many records to be have `records_number = 5`.
2) in Then `assert .count() == records_number)
3) all (check that data are as expected)